### ISSUE TITLE
`gsutil cp` creates subdirectory in bucket when executed from Windows

### ISSUE THREAD

**Issue by borekb** [GoogleCloudPlatform/gsutil#534]
Example command, produces exactly the same output on both Windows and Mac:

```
$ gsutil cp dist/command-port-042842b.zip gs://vp-builds/command-port/
Copying file://dist/command-port-042842b.zip [Content-Type=application/x-zip-compressed]...
- [1 files][  6.2 MiB/  6.2 MiB]
Operation completed over 1 objects/6.2 MiB.
```

On a Mac, this uploads the ZIP file as `gs://vp-builds/command-port/command-port-042842b.zip`.

On Windows, it uploads the file as `gs://vp-builds/command-port/dist/command-port-042842b.zip`, i.e., it creates an additional `dist` folder in GCS. I've tested it in both `cmd.exe` and Git Bash.

I'd argue that the behavior on Mac is the expected one and that Windows behavior is a bug.

gsutil version: 4.31

**Comment by houglum** [comment]
This is fixed as of v4.32, via this commit: https://github.com/GoogleCloudPlatform/gsutil/commit/56d1c1f6bb8eb6632683512f88148361665901df

**Comment by houglum** [comment]
Closing this, as #523 tracked the underlying issue and highlighted the naming mess it caused.
